browser/src/slideshow/Transition2d.ts

```diff
@@ -83,6 +83,7 @@ abstract class TransitionBase extends SlideChangeGl {
 class Transition2d extends TransitionBase {
 	private static readonly DefaultFromColor = new Float32Array([0, 0, 0, 0]);
 	private static readonly DefaultToColor = new Float32Array([0, 0, 0, 0]);
+	private _uniformCache: Record<string, WebGLUniformLocation> = {};
```

Comment:
I would suggest using the JavaScript built-in `Map` rather than the typescript `Record` (which is really just `Object`).

Reply:
solved on : https://github.com/CollaboraOnline/online/pull/11743/commits/4c9995ab6f6c255efcdad4ca0f46d7a7e9074545